app/assets/stylesheets/conversation_show.scss

```diff
@@ -32,16 +32,46 @@ margin: 0 0 5px 0;
   text-align: center;
 }
 
-.chatboxmessagecontent {
-  padding-left: 5px;
-  padding-bottom: 10px;
+// .chatboxmessagecontent {
+//   padding-left: 5px;
+//   padding-bottom: 10px;
+// }
+
+.chatboxmessagecontent time {
+    font-size: 9px;
+    color: #ccc;
 }
+
 .self {
-  color: blue;
+  justify-content: flex-end;
+  align-items: flex-end;
 }
 
-.other {
-  color: red;
+.other time{
+  position: absolute;
```

Comment:
Properties should be ordered border, border-bottom-color, border-left-color, clear, position